judgels-frontends/raphael/src/components/ProblemWorksheetCard/Bundle/ProblemStatementCard/ProblemStatementCard.tsx

```diff
@@ -58,7 +60,8 @@ export class ProblemStatementCard extends React.Component<ProblemStatementCardPr
           if (item.type === ItemType.Statement) {
             return this.renderStatement(item);
           } else {
-            return this.renderMultipleChoice(item);
+            multipleChoiceItemNum++;
+            return this.renderMultipleChoice(item, multipleChoiceItemNum);
```

Comment:
Wait, can this just use the new `item.number`? @nathanchrs ?

Reply:
Yes, `Item` have a `number` field now.

Reply:
@jauhararifin Please update

Reply:
Updated